docs/monthly-meeting/2024-01.md

```diff
@@ -0,0 +1,127 @@
+  - [CAM] Sphinx has ways to generate docs for a whole module, or just a single function, from docstrings. But yes, starting to use that would need a PEP.
+  - [CAM] The strategy we took for `sqlite3` was to focus on the functions with most arguments, where the bulleted lists have most benefit, and leave the smaller functions for later, when we have more experience with the approach.
+  - [Carol] And we should keep in mind that our perspectives are not necessarily the users' perspective. If we do make major changes we should think about users and we should vet the changes.
+  - Ryan started contributing by trying to get the contract of [`collections.Counter.most_common`](https://docs.python.org/3/library/collections.html#collections.Counter.most_common), and not finding it in the docstring nor in the documentation.
+  - Carol: If you're coming from the scientific world, where the `?` in IPython gives you the docstring, that is often the first thing you look at.
+  - Ryan will take a look at only documenting return type and "raises" in a structured way, leaving parameters in the prose.
+  - Guido warns that `collections.Counter` is not that typical.
+
+- If we have time, would like to hear from Daniele on the [thread he started on the structure of the Python docs](https://discuss.python.org/t/diataxis-and-python-documentation/41836) as to the takeways from that so far and next steps there.
+
+  - Discussion between Daniele and Ezio on the Python tutorial
+
+- [Ezio] The docs for builtin functions doesn't have any examples. Idea: Add a separate page with 1-3 examples for each function.
+- [Carol] always goes back to the users and what their pathways and needs are. The tutorial can't fit all users -- absolute beginners, people for whom Python is the first language, people coming from other languages. The current tutorial doesn't work for completely new users.
+- [CAM] The current tutorial targets people who are already familiar with programming languages. Perhaps we should have separate tutorials for complete beginners and people coming from specific languages.
+- [Daniele] One drawback of the current tutorial is lots of preamble saying why you want to learn Python. But the user is already here, wanting to learn Python. We should start directly with the examples. Also, we don't need to cover the edge cases right after people do something for the first time. Long explanations detract from the tutorial.
+- [Ezio] It seems that for the tutorial, examples work better than prose, for both new and experienced users. Also, we could have cheatsheets for people coming from other languages.
+- [Carol] There's a bigger pressing need for onboarding people from different groups, like people who don't know how to use the command line.
+
+
+## Follow-ups from previous meeting(s)
```

Comment:
We can remove this section, the bits we covered have been added above.

Reply:
I meant remove this section from this line down, not the preceding paragraph.

Reply:
"## Follow-ups from previous meeting(s)" ?

Reply:
Yes, starting from and including:

`## Follow-ups from previous meeting(s)`

Up to but excluding:

`## Next meeting`

And re-instate the deleted paragraph.